frontend/packages/noobaa-storage-plugin/integration-tests/views/noobaaDashboardPage.view.ts

```diff
@@ -0,0 +1,12 @@
+import { $, element, by } from 'protractor';
+
+export const obcCount = element(by.css('div[aria-label="Object Bucket Claim count"]'));
```

Comment:
`$()` is an alias for `element(by.css())`, so these can be simplified by just using `$()`.